src/jobs/build_docker.yml

```diff
@@ -8,6 +8,14 @@ parameters:
   image_name:
     type: string
     default: "quay.io/hmpps/${CIRCLE_PROJECT_REPONAME}"
+  container_registry_server:
+    description: >
+      The registry server to push the image to. This should be the same as the image_name
+      but without the image name. For example, if the image_name is
+      quay.io/hmpps/my-app, then the container_registry_server should be quay.io.
+      Alternatively this value might be, e.g. docker.io or ghcr.io.
```

Comment:
would be worth adding in that it uses `QUAYIO_USERNAME` and `QUAYIO_PASSWORD` for `quay.io` and `DOCKER_USERNAME` / `DOCKER_PASSWORD` for all other registries